minidump-processor/src/stackwalker/mod.rs

```diff
@@ -230,6 +230,27 @@ async fn instruction_seems_valid_by_symbols<P>(
 where
     P: SymbolProvider + Sync,
 {
+    // Our input is a candidate return address, but we *really* want to validate the address
+    // of the call instruction *before* the return address. In theory this symbol-based
+    // analysis shouldn't *care* whether we're looking at the call or the instruction
+    // after it, but there is one corner case where the return address can be invalid 
```

Comment:
```suggestion
    // after it, but there is one corner case where the return address can be invalid
```

Sorry, left a trailing whitespace!